zerver/tests/test_upload.py

```diff
@@ -1540,23 +1545,33 @@ def test_system_bot_avatars_url(self) -> None:
             settings.EMAIL_GATEWAY_BOT,
         ]
         internal_realm = get_realm(settings.SYSTEM_BOT_REALM)
+        default_avatar = DEFAULT_AVATAR_FILE
 
         for email in system_bot_emails:
             system_bot = get_system_bot(email, internal_realm.id)
             response = self.client_get(f"/avatar/{email}")
             redirect_url = response["Location"]
             self.assertEqual(redirect_url, str(avatar_url(system_bot)))
-            self.assertTrue(str(redirect_url).endswith(".png"))
-            self.assertFalse(str(redirect_url).endswith("unknown.png"))
 
             response = self.client_get(f"/avatar/{email}/medium")
             redirect_url = response["Location"]
             self.assertEqual(redirect_url, str(avatar_url(system_bot, medium=True)))
-            self.assertTrue(str(redirect_url).endswith("-medium.png"))
-            self.assertFalse(str(redirect_url).endswith("unknown-medium.png"))
```

Comment:
these are deleted because the tests below will cover that. This is added just incase we don't want the second commit (add checks to make sure these files exists)